pyrefinebio/dataset.py

```diff
@@ -61,10 +61,24 @@ def __init__(
 
     @classmethod
     def get(cls, id):
-        response = get_by_endpoint("dataset/" + id)
+        response = get_by_endpoint("dataset/" + id).json()
         return Dataset(**response)
 
 
+    def add_samples(self, experiment, samples=["ALL"]):
+        if self.is_processing or self.is_processed:
+            print("Cannot add samples to a dataset that has been processed!")
+            return
+
+        if self.data:
+            self.data = {**self.data, experiment: samples}
```

Comment:
Any reason to do this instead of `self.data[experiment] = samples`?

Also are `experiment` and `samples` objects or strings of accession codes here? It might actually make things easier to use if we support both seamlessly. (The dict we POST will need to be all strings though.)